internal/e2e/rds_test.go

```diff
@@ -2579,6 +2582,17 @@ func websocketroute(c string) *envoy_api_v2_route.Route_Route {
 	return cl
 }
 
+func headerroute(c string) *envoy_api_v2_route.Route_Route {
+	cl := routecluster(c)
+	//cl.Route_mat
```

Comment:
nit: looks like debugging breadcrumb

Reply:
Turns out the whole method isn't needed. =)